Glimps/tests/conftest.py

```diff
@@ -15,3 +25,60 @@ def symphony_storage():
 
     rmtree(constants.DATA_STORAGE)
     constants.DATA_STORAGE = original_storage
+
+
+@pytest.fixture(scope="session")
+def set_up_retrieve_analysis_action(token):
+    action = RetrieveAnalysis()
+    action.module.configuration = GLIMPSConfiguration(api_key=token, base_url="https://gmalware.ggp.glimps.re")
+    return action
+
+
+@pytest.fixture(scope="session")
+def set_up_search_analysis_action(token):
+    action = SearchPreviousAnalysis()
+    action.module.configuration = GLIMPSConfiguration(api_key=token, base_url="https://gmalware.ggp.glimps.re")
+    return action
+
+
+@pytest.fixture(scope="session")
+def set_up_wait_for_action(token, symphony_storage):
+    action = WaitForFile(data_path=symphony_storage)
+    action.module.configuration = GLIMPSConfiguration(api_key=token, base_url="https://gmalware.ggp.glimps.re")
+    return action
```

Comment:
Thx for the suggestion,
We added the fixture as you recommended, and we removed most of the "set up" fixture because they were adding unnecessary complexity.